Nukeeper.BitBucketLocal/BitbucketLocalRestClient.cs

```diff
@@ -83,26 +84,26 @@ private async Task<T> HandleResponse<T>(HttpResponseMessage response, [CallerMem
 
         public async Task<IEnumerable<Repository>> GetProjects([CallerMemberName] string caller = null)
         {
-            var response = await GetResourceOrEmpty<IteratorBasedPage<Repository>>("projects?limit=999", caller);
+            var response = await GetResourceOrEmpty<IteratorBasedPage<Repository>>("rest/api/1.0/projects?limit=999", caller);
```

Comment:
Extract this to a const please. I like that it's moved here, that way in the future we can build a new/updated client which target a different rest api version

Reply:
I can do that